perma_web/frontend/components/CreateLink.vue

```diff
@@ -227,5 +227,7 @@ onBeforeUnmount(() => {
 
     <CaptureError />
 
+    <UploadForm v-if="isStaff" />
```

Comment:
Love the idea of having a toggle to hide still-in-progress work. I wonder if `isStaff` is fine-tuned enough: it would be nice to be able to deploy this so that admins like Kristi and Clare could continue to do their work in prod, without seeing still-in-progress components in their UI. 

Maybe let's set up a second Django Waffle flag for whether to show in-progress components? And then we could toggle that as desired, either by adding users to a group, or etc. Does that sound good to you? I could help, if it would be appreciated. Here is the PR where I made the [vue-dashboard](https://github.com/harvard-lil/perma/pull/3482/files)

When installing Django Waffle, I tested accessing the flags from JS, and it worked great: https://waffle.readthedocs.io/en/stable/usage/javascript.html. IIRC, I used the [inline script](https://waffle.readthedocs.io/en/stable/usage/javascript.html#as-an-inline-script) option, and then checked the now-present global `waffle` object for flags. But, I remember that there is also a quick setup that lets you get flag info [as JSON](https://waffle.readthedocs.io/en/stable/usage/json.html), which maybe is more convenient for loading into the Vue store? No opinion from me!!

Reply:
Sure thing. Following the waffle documentation, I passed the `waffle` object to the global JavaScript scope in [e9ecbdd](https://github.com/harvard-lil/perma/pull/3539/commits/e9ecbdd2c250d37f343bfc2971e6417e1a5f329a). 

I also created a new const, `showDevPlayground` to replace `isStaff` in [0f2c482](https://github.com/harvard-lil/perma/pull/3539/commits/0f2c482180d6031bb042eff3b58a813d331d93a8). Right now, the value of `showDevPlayground` is tied to the value of the `vue-dashboard` flag. But my thinking is that once a new feature flag exists, we could tie it to that. 

Do you have any preference for the name of the new flag? Definitely not at all tied to `devPlayground` if it feels another name is more succinct or more apt.

Reply:
Love it. `showDevPlayground` sounds like a great name for the flag.

Reply:
Would you prefer to include a migration in this PR, rather than merging this and then creating a separate PR for that?

Reply:
Yeah, might as well keep it all in one place! Then you won't have to merge this with any placeholders.